hack/jenkins/e2e.sh

```diff
@@ -105,6 +105,17 @@ GCE_FLAKY_TESTS=(
     "allows\sscheduling\sof\spods\son\sa\sminion\safter\sit\srejoins\sthe\scluster" # file: resize_nodes.go, issue: #13258
     )
 
+# The following tests are known to be slow running (> 2 min), and are
```

Comment:
side note: we can set Ginkgo's slow spec threshold (https://github.com/onsi/ginkgo/blob/master/config/config.go#L46) to 2m instead of 5s to help us track these tests.